src/mixins/linkConfig.js

```diff
@@ -44,18 +44,19 @@ export default {
           line: { stroke: 'orange' },
           '.joint-highlight-stroke': { 'display': 'none' },
         });
-
         this.shapeView.showTools();
       } else {
         this.shape.attr({
           line: { stroke: 'black' },
         });
-
         this.shapeView.hideTools();
       }
     },
   },
   computed: {
+    shapeView() {
```

Comment:
I did this too on a pr I just merged into develop for association flows. But I put it at the bottom of computed. Just check if this tries to put it in twice before merging.

Reply:
Never mind. It was going to add a duplicate. I fixed it.